### ISSUE TITLE
Put deprecated zone name to time_zone_links

### ISSUE THREAD

**Issue by fuelen** [mathieuprog/tz_extra#5]
Timezone `Europe/Kiev` was renamed to `Europe/Kyiv`, but I cannot find any info about `Europe/Kiev` in the corresponding record:
```
> TzExtra.countries_time_zones() |> Enum.filter(& &1.time_zone == "Europe/Kyiv")  
[
  %{
    coordinates: "+5026+03031",
    country: %{code: "UA", name: "Ukraine"},
    dst_offset: 10800,
    dst_zone_abbr: "EEST",
    pretty_dst_offset: "+03:00",
    pretty_utc_offset: "+02:00",
    time_zone: "Europe/Kyiv",
    time_zone_links: [],
    utc_offset: 7200,
    zone_abbr: "EET"
  }
]
```

**Comment by mathieuprog** [comment]
Indeed Europe/Kiev is not a link as we can see here:
https://data.iana.org/time-zones/tzdb/europe
(search on Europe/Kyiv doesn't show any Link entry)

But it is rather marked as legacy in the backward file:
https://data.iana.org/time-zones/tzdb/backward

I guess you have Europe/Kiev in some data source and it is not to be found in the output of the functions in this library.

What do you think about adding `all_time_zone_links` map?
* `time_zone_links`: for display purposes: a developer may want to display a time zone with the links in eg a time zone picker, but would not want to display legacy time zone names (this case was by the way the motivation of writing this library).
* `all_time_zone_links`: for searching a time zone (eg from a data source) if not found as canonical time zone.

**Comment by fuelen** [comment]
> I guess you have Europe/Kiev in some data source and it is not to be found in the output of the functions in this library.

Yes, I noticed this after update of the library.

> What do you think about adding all_time_zone_links map?

Sounds good, but you better know the internals of the library and what is the best approach :)

**Comment by mathieuprog** [comment]
Will be released around Monday

**Comment by mathieuprog** [comment]
Since 2022f the zone links have been restructured and all put in "backwards".
https://mm.icann.org/pipermail/tz/2022-September/031942.html

So the idea of having two lists (time_zone_links vs all_time_zone_links) has been abandoned and all the links are in the time_zone_links list, as you actually initially requested.

Published in 0.23 with new 2022f data. Can you let me know that it works fine for you having the right time zone info?
Ty

**Comment by fuelen** [comment]
It works fine, thank you!
